modules/web/serverfarm/tests/e2e/max/main.test.bicep

```
@@ -21,7 +21,7 @@ param serviceShort string = 'wsfmax'
 param enableDefaultTelemetry bool = true
 
 @description('Optional. A token to inject into the name of each resource.')
-param namePrefix string = '[[namePrefix]]'
+param namePrefix string = 'ttete'
```

```suggestion
param namePrefix string = '[[namePrefix]]'
```

Beware, if you debug the script locally it can happen that the `finally` / `revert` doesn't run anymore ;)

---

Reminds me when we first developed this script

---

thanks, I didn't know this happened. this is the first time I'm using the test-modulelocally script and @ahmadabdalla didn't warm me about this behaviour :(

---

Maybe on purpose :D